blog_system/urls.py

```diff
@@ -0,0 +1,9 @@
+from django.contrib import admin
+from django.urls import path, include
+from debug_toolbar.toolbar import debug_toolbar_urls
```

Comment:
It seems like there is an issue with the import statement on this line. The correct import for django debug toolbar would be `from debug_toolbar import urls as debug_toolbar_urls`. Please correct it.